README.md

```diff
@@ -30,125 +29,15 @@ Features
-----------------------------
-
 For basic installation without extra dependencies, run
 
     pip install odl
 
-If you don't have administrator rights on your machine, you can install packages locally as a user:
-
-    pip install --user odl
-
-#### Extra dependencies
-
-Additional dependencies can be specified in square brackets, separated by commas (no spaces!):
-
-    pip install odl[dep1,dep2]
-
-Possible choices:
-
-- `all` : Install with all extra dependencies.
-- `show` : Install [matplotlib](http://matplotlib.org/) to enable displaying capabilities.
-- `fft` : Install [pyFFTW](https://pypi.python.org/pypi/pyFFTW) for fast Fourier transforms. Note that this requires the [FFTW](http://fftw.org/) library to be available on your system.
-  Note also that even without this dependency, FFTs can be computed with Numpy's FFT library.
-- `wavelets` : Install [PyWavelets](https://pypi.python.org/pypi/PyWavelets) for wavelet transforms.
-
-Installing from source
-----------------------
-This installation method is intended for developers who want to make changes to the code. It assumes that the [Git](http://www.git-scm.com/) version control system is available on your system.
-
-To get ODL, clone the repository with the command
-
-    git clone <repo_uri>
-
-where `<repo_uri>` is the clone link from the top navigation bar on the [ODL GitHub page](https://github.com/odlgroup/odl). No GitHub account is required for this step.
-
-For installation in a local user folder, enter the top-level directory of the cloned repository and run
-
-    pip install --user --editable .
-
-**Don't forget the "." (dot) at the end** - it refers to the current directory, the location from where `pip` is supposed to install ODL.
-
-We recommend the `--editable` option (can be shortened to `-e`) since it installs a link instead of copying the files to your Python packages location. This way local changes to the code (e.g. after a `git pull`) take immediate effect without reinstall.
-
-If you prefer a system-wide installation, leave out the `--user` option. To do this, you need administrator rights.
-
-#### Extra dependencies
-
-You can give additional dependencies separated by commas in square brackets right after the ".":
-
-    pip install --user -e .[dep1,dep2]
-
-The simplest choice is `all`, which pulls all dependencies available through `pip`. See above for a list of
-possible values.
-
-Developers have an additional choice `testing`, which installs all dependencies related to unit
-testing. See the next section.
-
-Running unit tests
-------------------
-Unit tests in ODL are based on [pytest](https://pypi.python.org/pypi/pytest), and coverage reports are created by the [coverage](https://pypi.python.org/pypi/coverage/) module. These packages are installed if you run
-
-    pip install --user -e .[testing]
-
-Now you can check that everything was installed properly by running
+You can check that everything was installed properly by running
```

Comment:
I don't think we should mention this without telling that extra dependencies are required. People will run this, see an error and think WTF. This is anyway more for developers - if we instead tell them to run `python -c "import odl"`, that would be easier to understand.